CMake/ctkMacroBuildQtPlugin.cmake

```diff
@@ -88,7 +88,7 @@ macro(ctkMacroBuildQtPlugin)
   # Wrap
   set(MY_QRC_SRCS "")
   if(CTK_QT_VERSION VERSION_GREATER "4")
-    qt5_wrap_cpp(MY_MOC_CPP ${MY_MOC_SRCS} TARGET ${lib_name} OPTIONS -DHAVE_QT5)
+    qt5_wrap_cpp(MY_MOC_CPP ${MY_MOC_SRCS} OPTIONS -DHAVE_QT5)
```

Comment:
What was the reason for removing `TARGET ${lib_name}`? We actually need it so that the `qt5_wrap_cpp` macro can extract the target specific compile definitions which are then passed to moc. Otherwise moc could see different code paths in certain scenarios.